startup.cmd

```diff
@@ -50,7 +50,7 @@ SET PYTHONPATH=%TABPY_ROOT%\tabpy-server;%TABPY_ROOT%\tabpy-tools;%PYTHONPATH%
 SET STARTUP_CMD=python tabpy-server\tabpy_server\tabpy.py
 IF [%1] NEQ [] (
     ECHO     Using config file at %TABPY_ROOT%\tabpy-server\tabpy_server\%1
-    SET STARTUP_CMD=%STARTUP_CMD% --config=%1
+    SET STARTUP_CMD=%STARTUP_CMD% --config=%TABPY_ROOT%\tabpy-server\tabpy_server\%1
```

Comment:
We need to update the docs to specify that this command line parameter will now need to be a relative path (since we are appending/prepending %  to it)
.../docs/server-config.md

Reply:
We can't do that as we adding limitation for users to keep all their config files in one folder and not being able to use them from a shared folder. User should be able to specify either absolute or relative path IMO